src/api/modules/utils.ts

```diff
@@ -244,3 +250,122 @@ export const resolveAppId = (appName: string, appVersion: string): Promise<strin
 }
 
 export const isLinked = propSatisfies<string, Manifest>(contains('+build'), 'version')
+
+// PS: All functions below was merged from the other utils.ts file
```

Comment:
No need for this comment on the file (it could be a comment on the github PR though)